scripts/e2e-test.sh

```diff
@@ -3,18 +3,65 @@
 set -eux
 
 SCRIPTDIR=$(dirname "$(realpath "$0")")
-REGISTRY=$1
 TESTS="install basic_volume_io"
+DEVICE=
+REGISTRY=
+TAG=
 
-# TODO: Add proper argument parser
-if [ -z "$REGISTRY" ]; then
-  echo "Missing parameter registry"
+help() {
+  cat <<EOF
+Usage: $(basename $0) [OPTIONS]
+
+Options:
+  --device <path>           Device path to use for storage pools.
+  --registry <host[:port]>  Registry to pull the mayastor images from.
+  --tag <name>              Docker image tag of mayastor images (default "ci")
+
+Examples:
+  $(basename $0) --registry 127.0.0.1:5000 --tag a80ce0c
+EOF
+}
+
+# Parse arguments
+while [ "$#" -gt 0 ]; do
+  case $1 in
+    -d|--device)
+      shift
+      DEVICE=$1
+      shift
+      ;;
+    -r|--registry)
+      shift
+      REGISTRY=$1
+      shift
+      ;;
+    -t|--tag)
+      shift
+      TAG=$1
+      shift
+      ;;
+    *)
```

Comment:
```suggestion
    -h|--help)
      help
      exit 0
    *)
```

For the sake of completeness...

Reply:
👍